packages/flutter/test/material/text_field_focus_test.dart

```diff
@@ -119,6 +119,8 @@ void main() {
     expect(tester.testTextInput.isVisible, isTrue);
 
     tester.testTextInput.hide();
+    final EditableTextState state = tester.state<EditableTextState>(find.byType(EditableText));
+    state.connectionClosed();
```

Comment:
This test is testing a non realistic situation.

In real app, closing the input connection will cause the textfield to be unfocus, tapping it again, the textfield will regain focus and open the input connection.

This works before because it relies on the bug that the selection does not update correctly in rendereditable. So everytime you tap, you receive a new selection which cause a selection changes and open the keyboard.